### PR TITLE
feat(proto): update notification states and events

### PR DESCRIPTION
**Pull request by philanton**

### CODE REVIEW

File: proto/operator.proto [hunk 1]
```diff
@@ -106,29 +106,50 @@ message Notification {
...
-  NOTIFICATION_TYPE_MARGIN_UPDATED = 7;
-}
-
-enum ChannelStatus {
-  CHANNEL_STATUS_UNSPECIFIED = 0;
-  CHANNEL_STATUS_OPENING = 1;
-  CHANNEL_STATUS_OPEN = 2;
-  CHANNEL_STATUS_CHALLENGING = 3;
-  CHANNEL_STATUS_CLOSED = 4;
+enum ClearingEvent {
```
Comment (nksazonov) [inline proto/operator.proto:109]: `CLEARING_EVENT_AGREED_ON_POSTFUND` and `CLEARING_EVENT_PROCESS_MARGIN_CALL` are missing.

File: proto/operator.proto [hunk 2]
```diff
@@ -106,29 +106,50 @@ message Notification {
...
+  CLEARING_EVENT_FINALIZE_SETTLEMENT = 12;
+  CLEARING_EVENT_FAILED_SETTLEMENT = 13;
+  CLEARING_EVENT_CHALLENGE = 14;
+  CLEARING_EVENT_FINALIZE = 15;
+  CLEARING_EVENT_WITHDRAW = 16;
+  CLEARING_EVENT_CLEAR_CHALLENGE = 17;
+  CLEARING_EVENT_CHALLENGE_TIMEOUT = 18;
+}
+
+enum ClearingState {
```
Comment (nksazonov) [inline proto/operator.proto:131]: `CLEARING_STATE_PROCESSED_MARGIN_CALL` is missing.

File: proto/operator.proto [hunk 3]
```diff
@@ -106,29 +106,53 @@ message Notification {
...
+  CLEARING_EVENT_UNSPECIFIED = 0;
+  CLEARING_EVENT_INSTANTIATED = 1;
+  CLEARING_EVENT_ACCEPTED = 2;
+  CLEARING_EVENT_FAILED = 3;
+  CLEARING_EVENT_INITIATOR_FUNDED = 4;
+  CLEARING_EVENT_RESPONDER_FUNDED = 5;
+  CLEARING_EVENT_POSTFUND_PROPOSED = 6;
+  CLEARING_EVENT_AGREED_ON_POSTFUND = 7;
+  CLEARING_EVENT_MOVE_TO_OPERATIONAL = 8;
+  CLEARING_EVENT_ISSUE_MARGIN_CALL = 9;
```
Comment (MaxMoskalenko) [inline proto/operator.proto:119]: Remove `ISSUE_MARGIN_CALL`

File: proto/operator.proto [hunk 4]
```diff
@@ -106,29 +106,53 @@ message Notification {
...
+  CLEARING_EVENT_PROCESS_MARGIN_CALL = 10;
+  CLEARING_EVENT_STARTED_SETTLEMENT = 11;
+  CLEARING_EVENT_ISSUE_POST_SETTLEMENT_MARGIN = 12;
+  CLEARING_EVENT_FINALIZE_SETTLEMENT = 13;
+  CLEARING_EVENT_FAILED_SETTLEMENT = 14;
+  CLEARING_EVENT_CHALLENGE = 15;
+  CLEARING_EVENT_FINALIZE = 16;
+  CLEARING_EVENT_WITHDRAW = 17;
+  CLEARING_EVENT_CLEAR_CHALLENGE = 18;
+  CLEARING_EVENT_CHALLENGE_TIMEOUT = 19;
```
Comment (MaxMoskalenko) [inline proto/operator.proto:129]: We have `TimeoutChallengeClearingEvent`. We should consider which variant will be better

**Review by nksazonov — APPROVED** [review]

**Pull request merged**